dockertest/subtestbase.py

```diff
@@ -139,6 +139,24 @@ def failif_ne(actual, expected, reason=None):
         spec = "{}: expected " + arg + "; got " + arg
         raise DockerTestFail(spec.format(reason, expected, actual))
 
+    @staticmethod
+    def failif_not_in(needle, haystack, description=None):
+        """
+        Convenience method for subtests to test for an expected substring
+        being contained in a larger string, e.g. to look for XYZ in a
+        command's stdout/stderr.
+
+        :param needle: the string you're looking for
+        :param haystack: the actual string, e.g stdout results from a command
+        :param description: description of haystack, e.g. 'stdout from foo'
+        """
+        if description is None:
+            description = 'string'
+        if needle in haystack:
```

Comment:
I don't remember exactly, but seem to recall that using ``needle.find(haystack) > -1`` is a lot faster when haystack is large.  Not a problem, just something that I seem to remember being called on once a long time ago.